CHANGELOG.md

```diff
@@ -21,10 +21,10 @@ Changes since v2.8
 - Allow organization owners to edit resources, forms, licenses and workflows in their own organization (#1893)
 - Show resources, forms, licenses and workflows from all organizations to organization owners (#1893)
 - API: comments are now optional for commands
-- API: comments can have attachments (#1928)
 
 ### Additions
 - Generating bare-bones PDFs from applications. This is a non-experimental feature. Fancier PDF generation is still experimental and can be enabled with a flag. (#2053)
+- Most actions can now have attachments in addition to comments (#1928)
```

Comment:
I think is a bit unclear until we implement the proper multiple attachments. Shouldn't this be about work in progress until that?

Reply:
It's a self-contained and consistent feature even without multiple attachment support in the UI. It could be shipped as-is (but we might want to make additions).

Reply:
reworded the changelog line